KosherCocoa/Library/Core/Solar/KCNOAACalculator.h

```diff
@@ -0,0 +1,164 @@
+//
+//  KCNOAACalculator.h
+//  KosherCocoa
+//
+//  Created by Elyahu on 1/23/23.
+//
+
+@import Foundation;
+#import "KCGeoLocation.h"
+#import "KCConstants.h"
+#import "KCAstronomicalCalculator.h"
+
+/** A class that uses the US National Oceanicnand Atmospheric Administration Algorithm to calculate sunrise and sunset. */
+
+NS_SWIFT_NAME(KCNOAACalculator)
```

Comment:
We want to remove the `KC` prefix from the name here for idiomatic bridging to Swift.

Reply:
Done

Reply:
The NS_SWIFT_NAME in the SunriseAndSunsetCalculator used KC as well. Not sure if you want to look into that.

Reply:
Good call. You can change that one too, if you'd like.